cmds/ps/ps.go

```diff
@@ -166,24 +158,6 @@ func (pT *ProcessTable) PrepareString() {
 	pT.fstring = fstring
 }
 
-func isPermutation(check string, ref string) bool {
-	if len(check) != len(ref) {
-		return false
-	}
-	checkArray := strings.Split(check, "")
-	refArray := strings.Split(ref, "")
-
-	sort.Strings(checkArray)
-	sort.Strings(refArray)
-
-	for i := range check {
-		if checkArray[i] != refArray[i] {
-			return false
-		}
-	}
-	return true
-}
-
 // For now, just read /proc/pid/stat and dump its brains.
 // TODO: a nice clean way to turn /proc/pid/stat into a struct. (trying now)
```

Comment:
This TODO was accomplished two years ago, I wrote this. Would be nice if this was removed.

Reply:
done.